src/language/fi.json

```diff
@@ -1229,6 +1229,13 @@
           "description": "Pyydetään laatijalta lisää tietoa.",
           "publish-button": "Lähetä tietopyyntö"
         },
+        "rfi-reply": {
```

Comment:
Samat avaimet pitäisi lisätä sv.jsoniin myös varmaan?

Reply:
joo - lokalisointia olen ylipäänsä tehnyt vain joiltakin osin

Reply:
näyttöjen sisältö pitää sopia tarkemmin että tuota kannattaa lokalisoida - muuten tulee ylimääräistä työtä

Reply:
näiden pitäis helpottua huomattavasti kun aletaan käyttämään sitä tällähetkellä PRssä olevaa scriptiä tuon sv.jsonin ylläpitoon.

Reply:
niin ja kun sv tiedosto laitetaan kuntoon vasta sitten kun näytöt on sovittu tarkemmin niin siitä ei tarvitse tämän osalta vielä välittää